src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Cells/__tests__/HealthCertificateQRCodeCellViewModelTests.swift

```diff
@@ -4,21 +4,86 @@
 
 import XCTest
 @testable import ENA
+import HealthCertificateToolkit
 
 class HealthCertificateQRCodeCellViewModelTests: XCTestCase {
 
-	func testGIVEN_HealthCertificateQRCodeCellViewModel_THEN_IsInitCorrect() throws {
+	func testGIVEN_ViewModelWithVaccinationCertificate_THEN_IsInitCorrect() throws {
 		// GIVEN
 		let viewModel = HealthCertificateQRCodeCellViewModel(
-			healthCertificate: HealthCertificate.mock(),
+			healthCertificate: try HealthCertificate(
+				base45: try base45Fake(
```

Comment:
How hard would it be to create the real base45 value?

Reply:
The creates the actual base45 value, just with fake data instead of a correctly signed certificate.